pip/cmdoptions.py

```diff
@@ -523,6 +524,58 @@ def only_binary():
 )
 
 
+def _good_hashes():
+    """Return names of hashlib algorithms at least as strong as sha256."""
+    # Remove getattr when 2.6 dies.
+    algos = set(
+        getattr(hashlib,
```

Comment:
This should probably get moved to ``pip/compat.py`` so we'd have like ``from pip.compat import hashlib_algorithms``